airlock/workspace_api.py

```diff
@@ -63,6 +63,30 @@ def get_url(self, relpath):
         )
 
 
+@dataclasses.dataclass(frozen=True)
+class OutputRequest(Container):
```

Comment:
Is it an OutputRequest or a ReleaseRequest?  I may just be still slightly confused about terminology

Reply:
Fair point.  I used OutputRequest because ReleaseRequest sounds worse to say out load :shrug:

Reply:
I have renamed.